src/sonic-host-services/scripts/hostcfgd

```diff
@@ -494,9 +511,11 @@ class AaaCfg(object):
         self.hostname = ""
 
     # Load conf from ConfigDb
-    def load(self, aaa_conf, tac_global_conf, tacplus_conf, rad_global_conf, radius_conf):
+    def load(self, aaa_conf, tac_global_conf, tacplus_conf, rad_global_conf, radius_conf, policies_conf):
```

Comment:
I will move the logic to a new class as you suggest.

The reason it was coded in that way is that in the beginning we thought that this feature touch the same file as aaa, so it was here in order to avoid conflict when rendering, but in the end the feature touched a different PAM file. so can be code separate.

Reply:
Done